docs/relational-databases/collations/collation-and-unicode-support.md

```diff
@@ -118,7 +118,7 @@ Each collation is combined as a series of suffixes to define case-, accent-, wid
 Windows collations define rules for storing character data that's based on an associated Windows system locale. For a Windows collation, you can implement a comparison of non-Unicode data by using the same algorithm as that for Unicode data. The base Windows collation rules specify which alphabet or language is used when dictionary sorting is applied. The rules also specify the code page that's used to store non-Unicode character data. Both Unicode and non-Unicode sorting are compatible with string comparisons in a particular version of Windows. This provides consistency across data types within [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], and it lets developers sort strings in their applications by using the same rules that are used by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. For more information, see [Windows Collation Name (Transact-SQL)](../../t-sql/statements/windows-collation-name-transact-sql.md).    
     
 #### <a id="Binary-collations"></a> Binary collations
-Binary collations sort data based on the sequence of coded values that are defined by the locale and data type. They're case-sensitive. A binary collation in [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] defines the locale and the ANSI code page that's used. This enforces a binary sort order. Because they're relatively simple, binary collations help improve application performance. For non-Unicode data types, data comparisons are based on the code points that are defined on the ANSI code page. For Unicode data types, data comparisons are based on the Unicode code points. For binary collations on Unicode data types, the locale isn't considered in data sorts. For example, **Latin1_General_BIN** and **Japanese_BIN** yield identical sorting results when they're used on Unicode data. For more information, see [Windows Collation Name (Transact-SQL)](../../t-sql/statements/windows-collation-name-transact-sql.md).   
+Binary collations sort data based on the sequence of coded values that are defined by the locale and data type. They're case-sensitive. A binary collation in [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] defines the locale and the ANSI code page that's used. This enforces a binary sort order. Because they're relatively simple, binary collations help improve application performance. For non-Unicode data types, data comparisons are based on the code points that are defined on the ANSI code page. For Unicode data types, data comparisons are based on the Unicode code points. For binary collations on Unicode data types, the locale is not considered in data sorts. For example, **Latin1_General_BIN** and **Japanese_BIN** yield identical sorting results when they're used on Unicode data. For more information, see [Windows Collation Name (Transact-SQL)](../../t-sql/statements/windows-collation-name-transact-sql.md).   
```

Comment:
```suggestion
Binary collations sort data based on the sequence of coded values that are defined by the locale and data type. They're case-sensitive. A binary collation in [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] defines the locale and the ANSI code page that's used. This enforces a binary sort order. Because they're relatively simple, binary collations help improve application performance. For non-Unicode data types, data comparisons are based on the code points that are defined on the ANSI code page. For Unicode data types, data comparisons are based on the Unicode code points. For binary collations on Unicode data types, the locale isn't considered in data sorts. For example, **Latin1_General_BIN** and **Japanese_BIN** yield identical sorting results when they're used on Unicode data. For more information, see [Windows Collation Name (Transact-SQL)](../../t-sql/statements/windows-collation-name-transact-sql.md).   
```